packages/lesswrong/components/comments/CommentsItem/CommentsItem.jsx

```diff
@@ -87,6 +87,8 @@ class CommentsItem extends Component {
 }
 
   replySuccessCallback = () => {
+    const { refetch } = this.props 
+    refetch()
```

Comment:
This is, effectively, making `refetch` a required argument, but not all the usages of `CommentsItem` pass it. In particular `CommentsList` and `RecentComments`.

Reply:
Also `recentDiscussionThreadsList`